wordpress-mobile/WordPress-iOS
# Remove Helpshift supporting code

Ref #9020 

This removes some code that was supporting Helpshift.
- `BadgeNavigationItemRow`.
- `WordPressSupportSourceTag+Helpers:toSupportSourceTag`

To test:
- Go to Me tab.
- Select Help & Support.
- Verify Support view is displayed.

![support_view](https://user-images.githubusercontent.com/1816888/41252355-1cd90650-6d7a-11e8-9968-91c4d7a1b4e3.png)
